File: app/src/main/java/com/gustu/logbook/main/model/levelKesulitan/Kesulitan.java
package com.gustu.logbook.main.model.levelKesulitan;

import com.google.gson.annotations.SerializedName;

public class Kesulitan {

	@SerializedName("RLK_KODE")
	private String rLKKODE;

	@SerializedName("RLK_NAMA")
	private String rLKNAMA;

	@SerializedName("RLK_USERUPDATE")
	private String rLKUSERUPDATE;

	@SerializedName("RLK_ISAKTIF")
	private String rLKISAKTIF;

	@SerializedName("RLK_LASTUPDATE")
	private String rLKLASTUPDATE;

	public void setRLKKODE(String rLKKODE){
		this.rLKKODE = rLKKODE;
	}

	public String getRLKKODE(){
		return rLKKODE;
	}

	public void setRLKNAMA(String rLKNAMA){
		this.rLKNAMA = rLKNAMA;
	}

	public String getRLKNAMA(){
		return rLKNAMA;
	}

	public void setRLKUSERUPDATE(String rLKUSERUPDATE){
		this.rLKUSERUPDATE = rLKUSERUPDATE;
	}

	public String getRLKUSERUPDATE(){
		return rLKUSERUPDATE;
	}

	public void setRLKISAKTIF(String rLKISAKTIF){
		this.rLKISAKTIF = rLKISAKTIF;
	}

	public String getRLKISAKTIF(){
		return rLKISAKTIF;
	}

	public void setRLKLASTUPDATE(String rLKLASTUPDATE){
		this.rLKLASTUPDATE = rLKLASTUPDATE;
	}

	public String getRLKLASTUPDATE(){
		return rLKLASTUPDATE;
	}

	@Override
 	public String toString(){
		return 
			"Kegiatan{" +
			"rLK_KODE = '" + rLKKODE + '\'' + 
			",rLK_NAMA = '" + rLKNAMA + '\'' + 
			",rLK_USERUPDATE = '" + rLKUSERUPDATE + '\'' + 
			",rLK_ISAKTIF = '" + rLKISAKTIF + '\'' + 
			",rLK_LASTUPDATE = '" + rLKLASTUPDATE + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/gustu/logbook/main/model/levelKesulitan/Kesulitan.kt
package com.gustu.logbook.main.model.levelKesulitan

import com.google.gson.annotations.SerializedName

class Kesulitan {

    @SerializedName("RLK_KODE")
    var rlkkode: String? = null

    @SerializedName("RLK_NAMA")
    var rlknama: String? = null

    @SerializedName("RLK_USERUPDATE")
    var rlkuserupdate: String? = null

    @SerializedName("RLK_ISAKTIF")
    var rlkisaktif: String? = null

    @SerializedName("RLK_LASTUPDATE")
    var rlklastupdate: String? = null

    override fun toString(): String {
        return "Kegiatan{" +
                "rLK_KODE = '" + rlkkode + '\''.toString() +
                ",rLK_NAMA = '" + rlknama + '\''.toString() +
                ",rLK_USERUPDATE = '" + rlkuserupdate + '\''.toString() +
                ",rLK_ISAKTIF = '" + rlkisaktif + '\''.toString() +
                ",rLK_LASTUPDATE = '" + rlklastupdate + '\''.toString() +
                "}"
    }
}